src/input.rs

```diff
@@ -1,5 +1,4 @@
-//! The [`Input`] trait that defines all possible inputs to [`cmd!`],
-//! [`cmd_unit!`] and [`cmd_result!`].
+//! The [`Input`] trait that defines all possible inputs to `cradle`.
```

Comment:
```suggestion
//! The [`Input`] trait that defines all possible inputs to a command.
```

TÌOLI

Reply:
I think 'command' is a bit weird for referring to child processes. Outside of Rust's standard library, I never think of a child process as a 'command'. And just using the word 'command' in lower-case, in natural English doesn't necessarily make me think of `std::process::Command`. (And -- strictly speaking -- `std::process::Command` is not a child process, but just the configuration for how to spawn one.)

But yeah, using `cradle` here is also not so nice. What do you think about saying '... inputs to a child process'?

Reply:
I think that sounds good.